src/rpc/server/scala/api/src/test/scala/com/ctc/omega_edit/SessionImplSpec.scala

```diff
@@ -158,6 +158,12 @@ class SessionImplSpec extends AnyWordSpec with Matchers with SessionSupport {
     }
   }
 
+  "profile" should {
+    "profile data" in session(as) { s =>
```

Comment:
With the way some other tests are made, like the one directly below this one, I would say instead of `"profile data"` do `"have data"`. I think that is a super small thing just seems like the way these tests are written.

Reply:
Maybe better it should say "profiler" should.